tools/verify-golint.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script checks coding style for go language files in each
+# Kubernetes package by golint.
+# Usage: `hack/verify-golint.sh`.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+#source "${KUBE_ROOT}/hack/lib/init.sh"
+#source "${KUBE_ROOT}/hack/lib/util.sh"
+
+#kube::golang::verify_go_version
+
+# Ensure that we find the binaries we build before anything else.
+KUBE_ROOT_ABSOLATE="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+
+export GOBIN="${KUBE_ROOT_ABSOLATE}/tools"
+PATH="${GOBIN}:${PATH}"
+
+# Install golint
+echo 'installing golint'
+pushd "${KUBE_ROOT}/tools" >/dev/null
+  GO111MODULE=on go get -u golang.org/x/lint/golint
+popd >/dev/null
+
+# check-file-in-alphabetical-order <file>
+# Check that the file is in alphabetical order
+#
+function check-file-in-alphabetical-order {
+  local failure_file="$1"
+  if ! diff -u "${failure_file}" <(LC_ALL=C sort "${failure_file}"); then
+    {
+      echo
+      echo "${failure_file} is not in alphabetical order. Please sort it:"
+      echo
+      echo "  LC_ALL=C sort -o ${failure_file} ${failure_file}"
+      echo
+    } >&2
+    false
+  fi
+}
+
+# arguments: target, item1, item2, item3, ...
+# returns 0 if target is in the given items, 1 otherwise.
+array_contains() {
```

Comment:
[minor] you use `function xyz { }` above and `xyz() { }` syntax, can we make it consistent